tcms/telemetry/static/telemetry/js/testing/breakdown.js

```diff
@@ -105,8 +146,16 @@ function drawChart(data, type, selector) {
         }
     };
     chartConfig.data = {
+        columns: chartData,
+        groups: groups,
         type: 'bar',
-        columns: values,
+        order: null
+    };
+    chartConfig.color = {
+        pattern: [
+            $.pfPaletteColors.blue,
+            $.pfPaletteColors.orange,
```

Comment:
Unconfirmed status should be `pf-red-100`.

Reply:
I choose that to be consistent with the automated/manual bar. will change to `pf-red-100`

Reply:
FYI visual consistency without consistency in meaning is not what we want!